frontend/src/app/assess/[projectId]/fairness-bias/api-history/[reportId]/page.tsx

```
@@ -0,0 +1,334 @@
+                                        <Server className="w-3.5 h-3.5" />
+                                        {report.config?.apiUrl}
+                                    </span>
+                                </div>
+                            </div>
+                        </div>
+
+                        <button
+                            onClick={exportPdf}
+                            disabled={isExporting}
+                            className="flex items-center gap-2 px-3 py-2 bg-primary/10 hover:bg-primary/20 text-primary border border-primary/20 rounded-lg transition-colors hide-in-pdf"
+                        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)